packages/distribution/addon/utils/inquiry-status.js

```diff
@@ -41,16 +43,49 @@ function decorator(
   return {
     get() {
       const inquiry = get(this, inquiryProperty);
-      const isAddressed = get(this, inquiryTypeProperty) === "addressed";
+      const inquiryType = get(this, inquiryTypeProperty);
+      const isAddressed = inquiryType === "addressed";
+      const isControlling = inquiryType === "controlling";
+
       const isDraft = isAddressed
         ? inquiry.status === "READY"
         : inquiry.status === "SUSPENDED";
       const isSent = !isAddressed && inquiry.status === "READY";
       const isSkipped = inquiry.status === "SKIPPED";
+      const isInProgress =
+        (isAddressed || isControlling) &&
+        inquiry.status === "READY" &&
+        inquiry.childCase?.document?.modifiedContentAt;
+
+      const inquiryAnswerStatus =
+        this.config.inquiry.status &&
+        (isAddressed || isControlling) &&
+        inquiry.status === "READY"
+          ? inquiry?.childCase?.workItems.edges
+              .filter((edge) => edge.node.status === "READY")
+              .map((edge) => {
+                const config = this.config.inquiry.status[edge.node.task.slug];
+                if (!config) {
+                  return null;
+                }
+
+                return {
+                  ...config,
+                  color: config.color[inquiryType] ?? config.color,
+                };
+              })
+              .filter(Boolean)[0]
+          : null;
+
+      if (inquiryAnswerStatus) {
+        return inquiryAnswerStatus;
+      }
```

Comment:
Where's the config part for this?

Reply:
Good point, I'll add one for the "confirm-inquiry-answer" work-item.

Reply:
Added